pom.xml

```diff
@@ -176,7 +176,7 @@
     <maven-checkstyle-plugin.version>3.3.0</maven-checkstyle-plugin.version>
     <maven-clover2-plugin.version>4.0.6</maven-clover2-plugin.version>
     <maven-pdf-plugin.version>1.6.1</maven-pdf-plugin.version>
-    <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
+    <maven-surefire-plugin.version>3.5.1</maven-surefire-plugin.version>
```

Comment:
Versions newer than 3.0.0-M4 are affected by https://issues.apache.org/jira/browse/SUREFIRE-1722

Maybe we have to decide which bug to live with ;)

Reply:
Good to know.  Let's remain in 3.0.0-M4 then.